packages/core/src/Stores/client-store.js

```diff
@@ -489,11 +490,11 @@ export default class ClientStore extends BaseStore {
         return false;
     }
 
-    @computed({ keepAlive: true })
+    @computed
     get is_eu_country() {
         const country = this.website_status.clients_country;
         if (country) return isEuCountry(country);
-        return country;
+        return false;
```

Comment:
@salar-fs I didn't get your point by returning `false` always for `is_eu_country`! is this a temporary situation? if yes, please add a TODO.

Reply:
@mahdiaryayi-fs It's not always `false`. If **`country`** exists (from website_status) and `isEuCountry` we return `true`. Otherwise it's `false`.